src/mutations/system/addBlockedSearchKeyword.ts

```diff
@@ -0,0 +1,27 @@
+import { UserInputError } from 'common/errors'
+import { MutationToAddBlockedSearchKeywordResolver } from 'definitions'
+
+const resolver: MutationToAddBlockedSearchKeywordResolver = async (
+  root,
+  { input: { keyword } },
+  { dataSources: { atomService, systemService }, viewer }
+) => {
+  const table = 'blocked_search_keyword'
+
+  // create
+  if (!keyword) {
```

Comment:
This is not necessary since schema [already did it](https://github.com/thematters/matters-server/pull/2668/files#diff-efc1675187620595b0844197a25c35eac9b6df752f9182e5cffddee6f27a8489R1820) (`!`).